test/presenters/return-versions/setup/additional-submission-options.presenter.test.js

```diff
@@ -39,7 +39,8 @@ describe('Return Versions Setup - Additional Submission Options presenter', () =
       expect(result).to.be.equal({
         backLink: `/system/return-versions/setup/${session.id}/check`,
         licenceId: '8b7f78ba-f3ad-4cb6-a058-78abc4d1383d',
-        additionalSubmissionOptions: [],
+        multipleUpload: undefined,
```

Comment:
I don't think this is correct anymore. `multipleUpload` is being set in `InitiateService`, so it will not be `undefined`.

We should probably update the 'session' object in this test (and other test files) to reflect this.

Reply:
Good spot i have done this else where (i think). 

Add multiple upload to default to false